whitelabelco/sofo-foods
# Cp/bug/missed style on merge conflict

### Description 💻

Missed a style change while resolving a merge conflict on previous PR

### Implementation 💡

Changes `justify-center` to `justify-start`

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- confirm landing hero text is aligned left

### Screenshots/Demos 📷



### Checklist ✅
